### PR TITLE
[welcomes] Fix and add some checks

### PR DESCRIPTION
**Pull request by kandnub**
Signed-off-by: Kandarp <Kandnub@gmail.com>

### CODE REVIEW

File: userbot/modules/welcomes.py [hunk 1]
```diff
@@ -56,7 +65,7 @@
 
                 # DEBUGGING. LEAVING IT HERE FOR SOME TIME ###
                 print(f"User Joined: {join_time}")
-                print(f"Spam Message Sent: {message_date}")
+                print(f"Message Sent: {message_date}")
```
Comment (baalajimaestro) [inline userbot/modules/welcomes.py]: Why is debugging stuff here? Doesn't it end up spamming console for every welcome mute applied

Comment (kandnub) [reply userbot/modules/welcomes.py]: It does but it says **LEAVING IT HERE FOR SOME TIME** so I left it for the person who added it.

Comment (baalajimaestro) [reply userbot/modules/welcomes.py]: I feel, you better nuke that. We just errors to be shown, not any random thing on terminal

Comment (kandnub) [reply userbot/modules/welcomes.py]: Nuked

**Pull request merged**